### DISCUSSION TITLE
It's seems that v0.4.8 produce more relevant images with same default hp than v0.4.9, v0.4.10.

### DISCUSSION THREAD

**Discussion by ZackPashkin** [lucidrains/big-sleep#28] [Ideas]
Hi! It's seems that v0.4.8 produce more relevant images with same default hp than v0.4.9, v0.4.10.  
Also predictions looks same, didn't change much, after first 30-100 iterations.
Any ideas how it can be improved

**Comment by DrJKL** [comment]
Hey Zack,

Do you have any examples?

https://github.com/lucidrains/big-sleep/compare/477b83234930d1d84d04ea34adbd790d9141f4ec...main

There shouldn't be any substantive differences in the generation between those versions.

**Reply by ZackPashkin** [reply]
Hi @DrJKL Here is some examples to compare: promt is  _**an apple next to a fireplace**_ Now seems 0.1.1 produced more relevant results on 1 epoch, comparing to 0.4.8, 0.4.9, 0.4.10
LEARNING_RATE = 5e-2
SEED = 0
![image](https://user-images.githubusercontent.com/39233070/107111970-b8301f80-6886-11eb-9b41-d138aa333b8d.png)

**Comment by ZackPashkin** [comment]
Seems ~ lr from 13e-2 to  20e-2  , iterations = 200 produce more semantically relevant results. What do you observe?
Also what do you think about prompt engineering . How long should be optimal length and should noun and adjective be duplicative

**Reply by LtqxWYEG** [reply]
I found that lr values over 0.11 appear to produce more non-converging garbage than values below. (But that is HEAVILY dependent on the seed) Also there are significant changes in the image up to 15k+ iterations. Just that they take longer to emerge
